src/algebra/category/Group/epi_mono.lean

```diff
@@ -0,0 +1,363 @@
+/-
+Copyright (c) 2022 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang
+-/
+import category_theory.epi_mono
+import group_theory.quotient_group
+import algebra.category.Group.basic
+
+/-!
+# Monomorphisms and epimorphism in `Group`
+In this file, we prove monomorphisms in category of group are injective homomorphisms and
+epimorphisms are surjective homomorphisms.
+-/
+
+
+universes u
+
+namespace monoid_hom
+
+open quotient_group
+
+variables {A B : Type u}
+
+section
+
+variables [group A] [group B]
+
+@[to_additive add_monoid_hom.ker_eq_bot]
+lemma ker_eq_bot (f : A →* B) :
```

Comment:
This already exists under the name `monoid_hom.ker_eq_bot_iff`.

Reply:
deleted